src/DataFixtures/UserFixtures.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\User;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+
+class UserFixtures extends Fixture
+{
+
+    public function load(ObjectManager $manager)
+    {
+        $user = new User();
+        $user->setUsername('John');
+
+        $password = "JohnPassword";
+        $user->setPlainPassword($password);
```

Comment:
Use the same syntax as the other Fixtures : 
`$user->setUsername('John')->setPassword('JohnPassword')->...`